Scripts/Player.gd

```diff
@@ -17,5 +17,5 @@ func _physics_process(delta):
 	else:
 		velocity = velocity.move_toward(Vector2.ZERO, FRICTION * delta)
 	
-	move_and_collide(velocity * delta);
+	move_and_slide(velocity);
```

Comment:
why? Do we need sliding at collision?

Reply:
So, both work pretty much the same, but:
move_and_collide has this little quirk where, when you collide with an object, your player sticks to that object and it gets harder to move away.
move_and_slide on the other hand, doesn't do that.
I hope this answer is acceptable.